python/tank/authentication/login_dialog.py

```diff
@@ -240,21 +240,21 @@ def __init__(
         self.ui.button_options.setVisible(False)
 
         self.menu_action_ulf2 = QtGui.QAction(
-            "Authenticate with your default web browser",
+            "Authenticate with App Session Launcher",
             menu,
         )
         self.menu_action_ulf2.triggered.connect(self._menu_activated_action_ulf2)
 
         self.menu_action_ulf = QtGui.QAction(
-            "Authenticate with the ShotGrid Desktop browser",
+            "Authenticate with the ShotGrid browser",
             menu,
         )
         self.menu_action_ulf.triggered.connect(self._menu_activated_action_web_legacy)
         menu.addAction(self.menu_action_ulf)
         menu.addAction(self.menu_action_ulf2)
 
         self.menu_action_legacy = QtGui.QAction(
-            "Authenticate with your login credentials",
+            "Authenticate with the classic credentials form",
```

Comment:
Comapring against some notes... should this be:
```suggestion
            "Authenticate with your legacy login credentials",
```